web/templates/settings/edit_custom_profile_field_form.hbs

```diff
@@ -36,6 +36,12 @@
         </div>
     </div>
     {{/if}}
+    {{> settings_checkbox
+      prefix="id-custom-profile-field-"
+      setting_name="editable-by-user"
+      is_checked=editable_by_user
+      label=(t "Users can edit this field")
+      }}
```

Comment:
It looks like you missed the feedback to move this setting last in the block?

Reply:
This is not ready for review yet. I only resolved the merge conflicts, I'm working on implementing the feedback now. 

Should be done in a couple of hours.

Reply:
Updated. Ready for review now.